spec/support/acts_as_tenant.rb

```diff
@@ -18,10 +18,17 @@ module ModelHelpers
     def with_tenant(tenant, &proc)
       context "with tenant #{tenant.inspect}" do |*params|
         before(:each) do
-          ActsAsTenant.current_tenant = send(tenant)
+          cur_tenant = send(tenant)
```

Comment:
use `current_tenant` instead of `cur_tenant`, follow the variable naming convention.

Reply:
Done